docs/runbooks/LowReadyVirtControllersCount.md

```diff
@@ -2,42 +2,44 @@
 
 ## Meaning
 
-From a high-level perspective the virt-controller has all the cluster wide virtualization functionality.
+This alert fires when one or more `virt-controller` pods are running, but none of these pods have been in a `Ready` state for the last 5 minutes. 
 
-This controller is responsible for monitoring the VMI (CRDs) and managing the associated pods. The controller will make sure to create and manage the life-cycle of the pods associated to the VMI objects.
+A `virt-controller` handles monitoring the custom resource definitions (CRDs) of a virtual machine instance (VMI) and managing the associated pods. The device creates pods for VMIs and manages the life-cycle of the pods.
 
-A VMI object will always be associated to a pod during it's life-time, however, due to i.e. migration of a VMI the pod instance might change over time.
-
-This alert is triggered when some virt controllers are running but not ready in the past 5 minutes.
+Therefore, `virt-controller` devices are critical for all cluster-wide virtualization functionality. 
 
 ## Impact
-In these circumstances the virt-controller will become a single point of failure.
+
+This alert indicates that a cluster-level failure may occur, which would cause actions related to VM life-cycle management to fail. This notably includes:
+
+- Launching a new VM instance
```

Comment:
I'm not sure this requires a bullet list. Also, you can use VMI, since you defined it above.

Suggestion: "... for example, launching a new VMI or shutting down an existing VMI."